src/main/scala/org/ergoplatform/modifiers/history/Header.scala

```diff
@@ -202,7 +175,7 @@ object HeaderSerializer extends Serializer[Header] {
 
     @tailrec
     def parseInterlinks(index: Int, endIndex: Int, acc: Seq[ModifierId]): Seq[ModifierId] = if (endIndex > index) {
-      val repeatN: Int = bytes.slice(index, index + 1).head
+      val repeatN: Int = 0xff & bytes(index)
```

Comment:
why not just bytes(index) ?

Reply:
`bytes(index)` will return signed byte, and this function will go to infinite loop if someone will eventually find a block with more then 127 leading zero bits.

Reply:
require(repeatN > 0) is not needed then, as your repeatN is non-negative by construction, or this line to be changed to just require(repeatN != 0)